### ISSUE TITLE
Link does not load when shared from another app

### ISSUE THREAD

**Issue by gokulraam** [uazo/bromite-buildtools#293]
Occurs if following settings is enabled
Clear browsing data at startup is enabled and all options checked.

Steps to reproduce:
1. The browser is actually closed
2. Share a link to the browser.
3. It opens about:blank page instead of the shared link.

**Comment by sankhababu** [comment]
> 

Yes it's happening. On the second attempt, without closing the browser, the link opens normally.

**Comment by uazo** [comment]
I'm trying it with v115 and it doesn't happen to me.
maybe you can try again as soon as I get it out.

**Comment by uazo** [comment]
please try to check with the latest version (v115)

**Comment by gokulraam** [comment]
There were two options shown in share menu for this browserq, one chromium and another open with bromite. Former does not work but latter option works.

**Comment by sankhababu** [comment]
Still happening, but not with all links. 
YouTube, Facebook links don't open in first time, and also https://worldarchitecture.org/architecture-news/cvepn/iitkharagpur-to-teach-vastu-shastra--an-8-000yearold-indian-architectural-science.html, https://mepaper.anandabazar.com/imageview_71066_45242450_4_71_28-06-2023_7_i_1_sf.html
with same problem. 

Not every time options like 'open with chromium and another open with bromite' shows.

**Comment by uazo** [comment]
> one chromium and another open with bromite. Former does not work but latter option works.

https://source.chromium.org/chromium/chromium/src/+/c0c799f0c471f9500a89a353a69f71d0bc57eda9
`open with chromium` is not my implementation but upstream! so theirs does not work properly.
I will delete it.
